trino/src/main/java/com/netease/arctic/trino/ArcticCatalogFactory.java

```diff
@@ -29,4 +29,6 @@ public interface ArcticCatalogFactory {
    * generate ArcticCatalog
    */
   ArcticCatalog getArcticCatalog();
+
+  ArcticConfig getArcticConfig();
```

Comment:
CatalogFactory has no responsibility to supply ArcticConfig. The ArcticConfig is a bean, you can directly inject it.